app/assets/src/styles/ui/controls/buttons/transparent_button.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../../themes/colors";
+@import "../../../themes/typography";
+
+.idseq-ui.button.transparent-btn {
+  border: 1px solid $white;
+  background-color: transparent;
+  color: $white;
+  height: 2rem;
+  min-width: 8rem;
+  margin-top: 0.4rem;
+
+  .icon-label {
+    svg {
+      fill: $white;
+    }
+  }
+
+  &:hover {
```

Comment:
Maybe make the color white with a small opacity? Would it work?

Reply:
you want the text color to be more transparent?

Reply:
Just the background. And i do not think the blue work well for black background and definitely not for all transparent buttons.